crates/cxx-qt-gen/src/parser/inherit.rs

```diff
@@ -5,28 +5,15 @@
 
 use crate::parser::method::MethodFields;
 use crate::parser::{check_safety, separate_docs};
-use crate::{
-    naming::Name,
-    parser::parameter::ParsedFunctionParameter,
-    syntax::{attribute::attribute_take_path, safety::Safety},
-};
+use crate::syntax::{attribute::attribute_take_path, safety::Safety};
 use quote::format_ident;
+use std::ops::Deref;
```

Comment:
prefer `core::ops::Deref` (eg prefer crates like alloc/core instead of std where possible). Then later then makes things easier to have no_std if ever possible in the future

Reply:
Done